pkg/minikube/download/docker_env.go

```diff
@@ -0,0 +1,42 @@
+package download
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+// DownloadNerdctl downloads nerdctl to directory specified by the param
+func DownloadNerdctl(dir string) error {
+	//check whether nerdctl exists
+	if _, err := os.Stat(filepath.Join(dir, "nerdctl")); err == nil {
+		//file exists
+		return nil
+	}
+	fileName := filepath.Join(dir, "nerdctl-1.0.0-linux-amd64.tar.gz")
+	if err := download("https://github.com/containerd/nerdctl/releases/download/v1.0.0/nerdctl-1.0.0-linux-amd64.tar.gz", fileName); err != nil {
```

Comment:
The version should be a variable so if we want to update the version we don't have to change it three times.